scripts/bench/stt-bench.py

```
@@ -0,0 +1,1508 @@
+    if isinstance(payload, list):
+        samples, meta = payload, {}
+    else:
+        samples, meta = payload.get("samples", []), {k: v for k, v in payload.items() if k != "samples"}
+    if not samples:
+        raise SystemExit(f"{manifest_path}: no samples in the manifest")
+    base = manifest_path.parent
+    resolved = []
+    for sample in samples:
+        for field in ("id", "lang", "audio", "reference"):
+            if field not in sample:
+                raise SystemExit(f"{manifest_path}: sample {sample.get('id', '?')} is missing '{field}'")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)